controller/integration_tests/stepper_test.h

```diff
@@ -0,0 +1,27 @@
+// This test rotates a stepper clockwise at a predefined rate and steps
+
+#include "hal.h"
+#include "stepper.h"
+#include <cmath>
+
+// test parameters
+static constexpr float step_degrees{-30.0f};
+static constexpr int64_t delay_ms{1000};
+
+void run_test() {
+  Hal.init();
+
+  // Configure stepper
+  StepMotor *stepper_motor = StepMotor::GetStepper(0);
+  stepper_motor->SetAmpAll(0.1f);
```

Comment:
It would probably be a good idea to make sure stepper_motor isn't NULL here.  That's unlikely, but would happen if the constant in stepper.h was set to zero.

Reply:
Failing is not necessarily bad. Could we make it fail interestingly somehow? Like, maybe it could send a signal on one of the pins that a testing environment could pick up on it, then it would be useful. We will definitely iterate on this.